lib/avirwrapper.php

```diff
@@ -72,7 +72,12 @@ function () use ($scanner, $path) {
 						$status = $scanner->completeAsyncScan();
 						if ($status->getNumericStatus() == \OCA\Files_Antivirus\Status::SCANRESULT_INFECTED){
 							$this->unlink($path);
-							throw new InvalidContentException($status->getDetails());
+							throw new InvalidContentException(
+								$this->l10n->t(
+									'The file is infected with %s virus. Upload terminated.',
```

Comment:
It won't be shown to WebDAV client because  but it is logged on server:
```
{"reqId":"x3VmeazQ7weIey\/ooscv","remoteAddr":"::1","app":"webdav","message":"Exception: {\"Message\":\"HTTP\\\/1.1 415 The file is infected with Eicar-Test-Signature virus. Upload terminated.\",\"Code\":0,\"Trace\":\"#0 \\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/lib\\\/private\\\/connector\\\/sabre\\\/file.php(150): OC\\\\Connector\\\\Sabre\\\\File->convertToSabreException(Object(OCP\\\\Files\\\\InvalidContentException))\\n#1 \\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/lib\\\/private\\\/connector\\\/sabre\\\/directory.php(113): OC\\\\Connector\\\\Sabre\\\\File->put(Resource id #271)\\n#2 \\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(1053): OC\\\\Connector\\\\Sabre\\\\Directory->createFile('inf65.txt', Resource id #271)\\n#3 \\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/CorePlugin.php(513): Sabre\\\\DAV\\\\Server->createFile('inf65.txt', Resource id #271, NULL)\\n#4 [internal function]: Sabre\\\\DAV\\\\CorePlugin->httpPut(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#5 \\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/3rdparty\\\/sabre\\\/event\\\/lib\\\/EventEmitterTrait.php(105): call_user_func_array(Array, Array)\\n#6 \\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(469): Sabre\\\\Event\\\\EventEmitter->emit('method:PUT', Array)\\n#7 \\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/3rdparty\\\/sabre\\\/dav\\\/lib\\\/DAV\\\/Server.php(254): Sabre\\\\DAV\\\\Server->invokeMethod(Object(Sabre\\\\HTTP\\\\Request), Object(Sabre\\\\HTTP\\\\Response))\\n#8 \\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/apps\\\/files\\\/appinfo\\\/remote.php(83): Sabre\\\\DAV\\\\Server->exec()\\n#9 \\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/remote.php(83): require_once('\\\/home\\\/deo\\\/publi...')\\n#10 {main}\",\"File\":\"\\\/home\\\/deo\\\/public_html\\\/oc-tmp\\\/lib\\\/private\\\/connector\\\/sabre\\\/file.php\",\"Line\":437}","level":4,"time":"2015-06-30T18:33:38+00:00","method":"PUT","url":"\/~deo\/oc-tmp\/remote.php\/webdav\/inf65.txt"}
```